### PR TITLE
Corrected typos and made certain things more concise.

### PR DESCRIPTION
**Pull request by sorasakurakitty**
I've made changes to README.md and a couple other files to correct typos, grammatical errors, and lack of conciseness.

### CODE REVIEW

**Comment by sorasakurakitty** [comment]
Commit [Correct typos in-code](https://github.com/TechUnRestricted/windiskwriter/pull/5/commits/aa892e52ad9cd7ef3c181ab6a9df6d6d4ab8283c) should be checked to make sure it doesn't break anything, as I don't have experience with Objective-C.

File: README.md [hunk 1]
```diff
@@ -206,7 +203,7 @@ UI is designed to be user-friendly and intuitive, so <strong>you can create boot
 <h2>Planned Changes</h2>
 <ul>
    <li>
-      📁 <strong>Add support for the individual partitions selection</strong><br>
+      📁 <strong>Add support for the individually selecting partitions</strong><br>
```
Comment (TechUnRestricted) [inline README.md:206]: I’m not sure about this correction. I could be wrong - I don’t know. But I feel like:
<pre>
📁 <strong>Add support for the individual partition selection</strong>
</pre>
 is more suitable here.

At the moment, the app only lets you select whole devices, so it recreates the Disk Scheme (MBR / GUID), but I want to add the feature to choose a partition from the device list, so you can avoid formatting the entire flash drive.

**Review by TechUnRestricted — CHANGES_REQUESTED** [review]
I appreciate your help with the Open Source project!<br>
My native language is not English, so some parts of the text might sound odd to a native speaker. I am <b>very grateful</b> for your corrections in this project, but I have some feedback, I would like you to make some changes before I merge them.

Thank you!

File: README.md [hunk 2]
```diff
@@ -215,30 +212,30 @@ UI is designed to be user-friendly and intuitive, so <strong>you can create boot
...
   <br>
    <li>
       🗜 <strong>Add support for splitting install.esd (compressed system image) files</strong><br>
          <sub>
-           Some Windows .iso’s, (usually repacks) use .esd system images for the best compression.<br>
-           But there can be some situations where even the .esd file is too large to fit into FAT32 partitions.<br>
-           At this moment, only install.wim images can be split. Splitting .esd requires wimlib to be updated.
+           Some Windows ISOs, (usually repacks) use .esd system images for better compression.<br>
+           But sometimes, even a .esd file is too large to fit into FAT32 partitions.<br>
+           Right now, WinDiskImager can only split install.wim images. Splitting .esd requires wimlib to be updated.
```
[the remark this answers is not in the document]
Comment (sorasakurakitty) [reply README.md]: Fixed. Dunno how that flew over my head lol

File: README.md (hunk 1 above)
Comment (sorasakurakitty) [reply README.md]: I had a typo here, I meant to write "Add support for individually selecting partitions".

"The individual partitions selection" is gramatically incorrect

**Comment by sorasakurakitty** [comment]
I have made a commit to my fork that should correct both issues. Hope it works

**Comment by sorasakurakitty** [comment]
Sorry if anything here was done wrong, I've never made a pull request before.

**Comment by TechUnRestricted** [comment]
>  I've never made a pull request before.  

Everything is fine. I’ve also never used the pull requests option before.

Thank you for your contribution! Have a nice day 👍

**Pull request merged**